demosplan/DemosPlanCoreBundle/Logic/Document/ParagraphExporter.php

```diff
@@ -136,6 +136,7 @@ public function generatePdf($procedureId, $title, $category)
 
         $templateVars['list'] = ['documentlist' => $documentList];
         $templateVars['procedure'] = $procedure;
+        $templateVars['listwidth'] = 17;
```

Comment:
please comment the magic number. Is the export always in the same format? No variable sometimes landscape sometimes not?

Reply:
done in https://github.com/demos-europe/demosplan-core/pull/3281/commits/b8911304ceecc97bdb7464ab9d1ba8a57f9ce823. And no in this case there are no export options it's always the same Format for the paragraphs